### PR TITLE
LC-69 Fix premium smiles for sc2tv

### LINKED ISSUE #69: Fix premium smiles for sc2tv

### PR DESCRIPTION
**Pull request by Hukutoss**
Fixed #69

### CODE REVIEW

File: modules/chats/sc2tv.py [hunk 1]
```diff
@@ -51,28 +51,17 @@ def opened(self):
     def closed(self, code, reason=None):
         log.info("Websocket Connection Closed Down")
 
-    def allow_smile(self, smile, user_id):
+    def allow_smile(self, smile, subs):
         allow = False
 
-        if smile['level'] == 0:
+        if smile['user'] is not None:
```
Comment (DeForce) [inline modules/chats/sc2tv.py]: if 'user' in smile:

File: modules/chats/sc2tv.py [hunk 2]
```diff
@@ -51,28 +51,17 @@ def opened(self):
     def closed(self, code, reason=None):
         log.info("Websocket Connection Closed Down")
 
-    def allow_smile(self, smile, user_id):
+    def allow_smile(self, smile, subs):
```
Comment (DeForce) [inline modules/chats/sc2tv.py]: def allow_smile(self, smile, subscriptions):

**Comment by Hukutoss** [comment]
Updated

**Comment by DeForce** [comment]
squash

**Pull request merged**